src/index.html

```diff
@@ -22,6 +22,316 @@
+            <div class="stars__star-1"></div>
+            <div class="stars__star-2"></div>
+            <div class="stars__star-3"></div>
+            <div class="stars__star-4"></div>
+            <div class="stars__star-5"></div>
+          </div>
+          Reviews: 5
+        </div>
+        <div class="card__price">Price:</div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__picture"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star-1"></div>
+            <div class="stars__star-2"></div>
+            <div class="stars__star-3"></div>
+            <div class="stars__star-4"></div>
+            <div class="stars__star-5"></div>
+          </div>
+          Reviews: 5
+        </div>
+        <div class="card__price">Price:</div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__picture"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star-1"></div>
+            <div class="stars__star-2"></div>
+            <div class="stars__star-3"></div>
+            <div class="stars__star-4"></div>
+            <div class="stars__star-5"></div>
+          </div>
+          Reviews: 5
+        </div>
+        <div class="card__price">Price:</div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__picture"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star-1"></div>
+            <div class="stars__star-2"></div>
+            <div class="stars__star-3"></div>
+            <div class="stars__star-4"></div>
+            <div class="stars__star-5"></div>
+          </div>
+          Reviews: 5
+        </div>
+        <div class="card__price">Price:</div>
+        <a
+          class="card__button"
+          href="#"
+        >
+          BUY
+        </a>
```

Comment:
The product cards are repeated multiple times with the same content. Consider using a template or a loop in your JavaScript to dynamically generate these cards if the content is the same. This will make your code more maintainable and easier to update.